src/components/Drawer/components/Drawer.scss

```diff
@@ -0,0 +1,187 @@
+@use '../../variables';
+
+$block: '.#{variables.$ns}drawer';
+$drawer-resizer-width: 8px;
+
+#{$block} {
+    pointer-events: none;
+    outline: none;
+
+    &__item {
+        position: absolute;
+        z-index: 110;
```

Comment:
Let's keep it high enough. It should be equal or maybe slightly less than Modal's zIndex